src/utils/FileUtil.js

```diff
@@ -1,82 +1,58 @@
-var fs = require('fs');
-var glob = require('glob');
-var UglifyJS = require('uglifyjs');
+import UglifyJS from "uglifyjs"
```

Comment:
here make sense to use uglify-js ? instead of uglifyjs which seems to be deprecated